src/module/rent/domain/rent.entity.ts

```diff
@@ -0,0 +1,53 @@
+import Car from "../../car/domain/car.entity";
+import Customer from "../../customer/domain/customer.entity";
+
+export enum StatusEnum {
+	finished = "finished",
+	pending = "pending",
+	canceled = "canceled",
+}
+export class Rent {
+	id?: number;
+	carId: number;
+	customerId: number;
+	unitPrice: number;
+	totalPrice: number;
+	paymentMethod: string;
+	isPaid: boolean;
+	startDate: Date;
+	endDate: Date;
+	createdAt?: Date;
+	updatedAt?: Date;
+	car?: Car;
+	customer?: Customer;
+	status?: StatusEnum;
+
+	available(): boolean {
```

Comment:
Let's add a test to this domain entity, to make sure this method always work the right way

Reply:
ok